cmd/gwi-api/api/stores/postgrestore/migrations/sql/bindata.go

```diff
@@ -0,0 +1,281 @@
+// Code generated by go-bindata.
+// sources:
+// 1_init.down.sql
+// 1_init.up.sql
+// sql.go
+// DO NOT EDIT!
+
+package sql
+
+import (
+	"bytes"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+func bindataRead(data []byte, name string) ([]byte, error) {
```

Comment:
Hmm, I'm not familiar with `go-bindata` but ... is this something that should go to source control?

Reply:
It is generated code. You do commit it. It would be a pain to expect anybody building the package to install the command line tool that generated this and do it themselves. I mean, it is certainly possible, but it is easier to do it once and commit. You do the same thing when using gRPC or Protocol Buffers, right?

Reply:
I guess. And if I wanted to generate a new migration, wouldn't I need to build and install that command line tool anyway?

Reply:
Sure, but that happens during development, not testing or deployment. Like, when using a CI server, you would not need to install that tool there.

Reply:
I agree that it does not have to be included and used during tests on CI and deployments, however I also think that the tool should be available in the development docker image so that nobody needs to install anything extra in their day-to-day work.

Reply:
Yeah, I answer this in the other thread, sorry.